internal/runbits/errors/errors.go

```diff
@@ -32,6 +33,19 @@ type OutputError struct {
 	error
 }
 
+var errorId string
+
+// Returns a relatively unique ID for error reporting.
```

Comment:
We don't need a totally unique ID, just something to search for in Rollbar to find the user's log.